core/src/main/java/io/aiven/klaw/service/SchemaRegistrySyncControllerService.java

```diff
@@ -126,6 +131,8 @@ public SyncSchemasList getSchemasOfEnvironmentFromMetadataDb(
           schemaInfoList.stream().filter(schema -> schema.getTeamId() == teamId).toList();
     }
 
+    schemaInfoList = annotateSchemasNotInCluster(schemaInfoList, schemaEnvId, tenantId);
```

Comment:
```suggestion
    annotateSchemasNotInCluster(schemaInfoList, schemaEnvId, tenantId);
```
as it's already assigned